src/etc/inc/captiveportal.inc

```diff
@@ -1082,6 +1082,34 @@ function captiveportal_disconnect_client($sessionid, $term_cause = 1, $logoutRea
 	}
 }
 
+/* remove all clients */
+function captiveportal_disconnect_all($term_cause = 6, $logoutReason = "DISCONNECT") {
+	global $g, $config, $cpzone, $cpzoneid;
+
+	$radiussrvs = captiveportal_get_radius_servers();
+	$cpdb = captiveportal_read_db();
+
+	$unsetindexes = array();
+
+	$stop_time = time();
+
+	foreach ($cpdb as $cpentry) {
+		if (empty($cpentry[11])) {
+			$cpentry[11] = 'first';
+		}
+		$radiusservers = $radiussrvs[$cpentry[11]];
+
+		captiveportal_disconnect($cpentry, $radiusservers, $term_cause, $stop_time);
```

Comment:
The idea was to ensure that the stop time was the same for all the sessions, since, in theory, if there were hundreds or thousands of active sessions, it could change between the first and the last one. Thinking about it again it's probably better to just remove it and use the real disconnection time, I'll change it.